src/bin/index.ts

```diff
@@ -8,19 +8,25 @@ import { initialize } from "../initialize/index.js";
 import { migrate } from "../migrate/index.js";
 import { logLine } from "../shared/cli/lines.js";
 import { StatusCodes } from "../shared/codes.js";
+import { logHelpText } from "./help.js";
 import { promptForMode } from "./mode.js";
+import { getVersionFromPackageJson } from "./packageJson.js";
 
 const operationMessage = (verb: string) =>
 	`Operation ${verb}. Exiting - maybe another time? 👋`;
 
 export async function bin(args: string[]) {
 	console.clear();
 
+	const version = await getVersionFromPackageJson();
+
 	prompts.intro(
 		[
-			chalk.greenBright(`Welcome to`),
+			chalk.greenBright(`✨ Welcome to`),
 			chalk.bgGreenBright.black(`create-typescript-app`),
-			chalk.greenBright(`! 🎉`),
+			chalk.greenBright(
+				`${typeof version === "string" ? ` ${version}` : ""}! ✨`,
+			),
```

Comment:
```plaintext
┌  ✨ Welcome to create-typescript-app  1.29.39! ✨
```

[Cleanup] Looks like there's an extra space before the version number?

Reply:
You're right! Fixed.